Client/src/Pages/Incidents/index.jsx

```diff
@@ -1,9 +1,72 @@
-import React from 'react'
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import StatusLabel from "../../Components/Label/";
 
-const Incidents = () => {
-    return (
-        <div>Incidents</div>
-    )
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.action.hover, 
+    color: theme.palette.common.black,
+  },
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(1)': {
+    backgroundColor: theme.palette.action.hover, 
+  },
+  '&:nth-of-type(n+1)': {
+    backgroundColor: theme.palette.common.white, 
+  },
+  '&:last-child td, &:last-child th': {
+    border: 0,
+  },
+}));
+
+function createData(name, calories, fat, carbs, protein) {
+  return { name, calories, fat, carbs, protein };
```

Comment:
I guess this is mock data for now so not a big deal, but the names definitely don't make sense in this context :joy: